pyfmt/__init__.py

```diff
@@ -47,11 +163,12 @@ def run_formatter(cmd, path, **kwargs) -> int:
     result = subprocess.run(cmd, stdout=PIPE, stderr=PIPE)
 
     prefix = f"{cmd[0]}: "
-    sep = "\n" + (" " * len(prefix))
+    display_divider(title=prefix[:-2])
+    sep = "\n"
     lines = result.stdout.decode().splitlines() + result.stderr.decode().splitlines()
     if "".join(lines) == "":
-        print(f"{prefix}No changes.")
```

Comment:
why remove the prefixes?

Reply:
I added a divider of each module's work, didn't think the indentation is necessary
...BUT this is completely subjective!  I'm ok with either way

Reply:
the prefix isn't indentation, it's just `isort: ` or `black: ` so you know which tool the output is from

Reply:
no i get that, prefix is still there, but in the divider now, but i removed the prefix as in line with the indentation like it was

Reply:
no i get that, the prefix is still there but moved to the divider.  I rembed the indentation part and the part where the prfix is in line with the indent